test/unit-test/main.tf

```diff
@@ -152,6 +152,87 @@ data "aws_iam_policy_document" "instance-scheduler-lambda-function-policy" {
 resource "aws_lambda_invocation" "test_invocation" {
   function_name = module.module_test.lambda_function_name
 
+  input = jsonencode(
+    {
+      action = "Test"
+  })
+}
+
+# lambda module test with zip package type and vpc config
+module "lambda_function_in_vpc" {
+  source           = "../../"
+  application_name = local.application_name
+  description      = "lambda function provisioned within a vpc test"
+  package_type     = "Zip"
+  filename         = data.archive_file.lambda-zip.output_path
+  source_code_hash = data.archive_file.lambda-zip.output_base64sha256
+  handler          = "test.lambda_handler"
+  runtime          = "python3.8"
+  tags             = local.tags
+  function_name    = "lambda-function-in-vpc-test"
+  create_role      = false
+  lambda_role      = aws_iam_role.lambda-vpc-role.arn
+
+  vpc_subnet_ids         = [data.aws_subnet.private-2a.id]
+  vpc_security_group_ids = [aws_security_group.lambda_security_group_test.id]
+}
+
+data "aws_iam_policy_document" "lambda_assume_role_policy" {
+  statement {
+    effect  = "Allow"
+    actions = ["sts:AssumeRole"]
+
+    principals {
+      type        = "Service"
+      identifiers = ["lambda.amazonaws.com"]
+    }
+  }
+}
+
+resource "aws_iam_role" "lambda-vpc-role" {
+  name = "LambdaFunctionVPCAccess"
+  tags = local.tags
+
+  assume_role_policy = data.aws_iam_policy_document.lambda_assume_role_policy.json
+}
+
+resource "aws_iam_role_policy_attachment" "lambda-vpc-attachment" {
+  role       = aws_iam_role.lambda-vpc-role.name
+  policy_arn = "arn:aws:iam::aws:policy/service-role/AWSLambdaVPCAccessExecutionRole"
+}
+
+data "aws_vpc" "platforms-test" {
+  id = "vpc-05900bb7e2e82391f"
+}
+
+data "aws_subnet" "private-2a" {
+  id = "subnet-0e2a4d5f4b346c981"
+}
+
+resource "aws_security_group" "lambda_security_group_test" {
+  name        = "lambda-vpc-module-test"
+  description = "lambda attached to vpc test security group"
+  vpc_id      = data.aws_vpc.platforms-test.id
+
+  egress {
+    description = "Allow all outbound traffic"
+    from_port   = 0
+    to_port     = 0
+    protocol    = "-1"
+  }
+
+  tags = local.tags
```

Comment:
Please move the argument on this line to be next to the other arguments.

> For blocks that contain both arguments and "meta-arguments" (as defined by the Terraform language semantics), list meta-arguments first and separate them from other arguments with one blank line. Place meta-argument blocks last and separate them from other blocks with one blank line.[^1]

[^1]: https://developer.hashicorp.com/terraform/language/syntax/style